webapp/sources/rudder/rudder-web/src/main/scala/bootstrap/liftweb/RudderConfig.scala

```diff
@@ -865,15 +865,52 @@ object RudderParsedProperties {
   }
 
   // don't parse some elements in inventories: processes
-  val INVENTORIES_IGNORE_PROCESSES:                   Boolean = {
+  val INVENTORIES_IGNORE_PROCESSES: Boolean = {
     try {
       config.getBoolean("inventory.parse.ignore.processes")
     } catch {
       case ex: ConfigException => false
     }
   }
+
+  val INVENTORIES_MAX_BEFORE_NOW: Duration = {
+    try {
+      Duration.fromScala(
+        scala.concurrent.duration.Duration.apply(
+          config.getString(
+            "inventories.reject.maxAgeBeforeNow"
+          )
+        )
+      )
+    } catch {
+      case ex: Exception =>
+        ApplicationLogger.error(
+          s"Error when reading key: 'inventories.reject.maxAgeBeforeNow', defaulting to 2 days: ${ex.getMessage}"
+        )
+        2.days
+    }
+  }
+
+  val INVENTORIES_MAX_AFTER_NOW: Duration = {
+    try {
+      Duration.fromScala(
+        scala.concurrent.duration.Duration.apply(
+          config.getString(
+            "inventories.reject.maxAgeAfterNow"
+          )
+        )
+      )
+    } catch {
+      case ex: Exception =>
+        ApplicationLogger.error(
+          s"Error when reading key: 'inventories.reject.maxAgeAfterNow', defaulting to 12h: ${ex.getMessage}"
+        )
```

Comment:
Is see that we usually log at `INFO` level when there is a `ConfigException`, and maybe we want a `WARN` level when the duration constructor throws a `NumberFormatException` because it is not causing an application logic error per se, we have a default value